src/presenter/general-presenter.js

```diff
@@ -16,19 +18,24 @@ export default class GeneralPresenter {
   #events = null;
   #sortView = null;
   #activeSortType = SortType.DAY;
+  #activeFilterType = FilterType.EVERYTHING;
   #filterView = null;
+  #noEventComponent = null;
 
   constructor({tripControlsFilters,tripEvents,pointModel}) {
     this.#tripControlsFilters = tripControlsFilters;
     this.#tripEvents = tripEvents;
     this.#pointModel = pointModel;
     this.#eventListComponent = new EventListView();
+    this.#pointModel.addObserver(this.#handleModelEvent);
+
   }
 
   init() {
-    this.#events = sortEvents(this.#pointModel.events,this.#activeSortType);
+    this.#events = filterEvents(this.#pointModel.events,this.#activeFilterType);
+    // this.#events = sortEvents(this.#pointModel.events,this.#activeSortType);
```

Comment:
У тебя не работает одновременно сортировка и фильтрация, потому что в обе функции ты подаешь эвенты из модели (т.е. исходные, неизмененные). А сортировка нужно эвенты, которые уже отфильтровали

```suggestion
    this.#events = filterEvents(this.#pointModel.events,this.#activeFilterType);
    this.#events = sortEvents(this.#events,this.#activeSortType);
```